src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/ControlDesigner.cs

```diff
@@ -289,11 +288,15 @@ public virtual SelectionRules SelectionRules
     internal Point GetOffsetToClientArea()
     {
         Point nativeOffset = default;
-        PInvoke.MapWindowPoints(Control, Control.Parent, ref nativeOffset);
+        if (Control.Parent is not null)
```

Comment:
Please try to use pattern matching wherever you can.

```suggestion
        if (Control.Parent is {} parent)
```